src/components/summary/_macro.njk

```diff
@@ -1,6 +1,8 @@
 {% macro onsSummary(params) %}
     {% set className = "ons-summary" %}
     {% set titleSize = "2" %}
+    {% set cardVariant =  params.variant if params.variant == "card" and not params.hub else ""%}
+    {% set cardVariantwithMargin = "--card ons-u-mb-l" if params.variant == "card" and not params.hub else ""%}
```

Comment:
`cardVariantwithMargin` isnt being used

Reply:
It should be removed apologies for the confusion